NotificationService/NotificationService.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NotificationService.swift
+//  NotificationService
+//
+//  Created by yagiz on 3/29/20.
+//  Copyright © 2020 Yagiz Nizipli. All rights reserved.
+//
+
+import UserNotifications
+import OneSignal
+
+class NotificationService: UNNotificationServiceExtension {
+    
+    var contentHandler: ((UNNotificationContent) -> Void)?
+    var receivedRequest: UNNotificationRequest!
```

Comment:
Bu neden force unwrapped ama diğer ikisi değil anlamadım. Sanki üçü de force unwrapped olmamalı gibime geldi çünkü didReceive içindeler. didReceive gelmeyebilir (adı üstünde didReceive, yani daha gelmediği bir state bulunuyor anlamına geliyor). Bu durumda birisi receivedRequest'e erişmeye çalışırsa crash olur. Bi daha kontrol eder misin?

Reply:
default onesignal'in onerdigi koddu bu, cok uzerine dusmedim.